bin/migrate.sh

```diff
@@ -50,32 +54,48 @@ echo -e "${YELLOW}************************${NC}"
 # echo -e "${YELLOW}Logs are being redirect to:${NC} \n\tQueries:\t${DUMPLOG}\n\tErrors:\t\t${ERRLOG}${NC}"
 
 echo -en "${YELLOW}Creating db${NC}..."
-psql --set ON_ERROR_STOP=on -U ${USER} -p ${PORT} -o ${ERRLOG} -c "DROP DATABASE IF EXISTS ${DB_NAME}" || error "Dropping old db"
-psql --set ON_ERROR_STOP=on -U ${USER} -p ${PORT} -o ${ERRLOG} -c "CREATE DATABASE ${DB_NAME}" || error "Creating db"
+psql --set ON_ERROR_STOP=on -h ${HOST} -U ${USER} -p ${PORT} -o ${ERRLOG} -d postgres -c "DROP DATABASE IF EXISTS \"${DB_NAME}\"" || error "Dropping old db"
+psql --set ON_ERROR_STOP=on -h ${HOST} -U ${USER} -p ${PORT} -o ${ERRLOG} -d postgres -c "CREATE DATABASE \"${DB_NAME}\"" || error "Creating db"
 echo "Done ($SECONDS)"
 
 SECONDS=0
 SQL="./output/${DB_NAME}/psql_tables.sql"
 echo -en "${YELLOW}Creating v2 tables${NC}..."
-psql --set ON_ERROR_STOP=on -U ${USER} -p ${PORT} -f ${SQL} -o ${ERRLOG} -d ${DB_NAME} || error "Creating tables"
+psql --set ON_ERROR_STOP=on -h ${HOST} -U ${USER} -p ${PORT} -f ${SQL} -o ${ERRLOG} -d ${DB_NAME} || error "Creating tables"
 echo "Done ($SECONDS)"
 
 SECONDS=0
+if [ ${DB_NAME} = "blpcore" ]
```

Comment:
Would you mind to explain what you are trying to achieve with this? As far as I infer you are doing some content parsing on the generated export. In case that is the case you should have used the [`mysql_raw_dump.json`](https://github.com/ggarri/mysql2psql/blob/master/rules/mysql_raw_dump.json) rules which are meant to customize the way raw column data is exported from mysql

Reply:
This commit is very case specific, I shouldn't have suggested it.